app/services/config_report_importer.rb

```diff
@@ -0,0 +1,21 @@
+class ConfigReportImporter < ReportImporter
+  attr_reader :report
+  def self.authorized_smart_proxy_features
+    ['Puppet']
+  end
+
+  def report_name_class
+    ConfigReport
+  end
+
+  private
+
+  def create_report_and_logs
+    @report = super
```

Comment:
@domcleal - this is weird.
It works on foreman as expected, yet tests fail with:
```ruby
test_0003_Report has child classes(ReportTest::Report STI):
NoMethodError: undefined method `persisted?' for nil:NilClass
    /home/szadok/dev/upstream/foreman/app/services/config_report_importer.rb:17:in `create_report_and_logs'
```
Any clue how to tackle this? (thanks)